magellan-library/src/main/java/com/wealthfront/magellan/transitions/CircularRevealTransition.kt

```diff
@@ -19,6 +19,12 @@ import kotlin.math.hypot
  */
 public class CircularRevealTransition(private val clickedView: View) : MagellanTransition {
 
+  private var animator: Animator? = null
+
+  override fun interrupt() {
+    animator?.end()
```

Comment:
sadly wasn't able to figure out a unit test for this. Couldn't get Robolectric to consider the necessary views attached